telemetry/definitions/commonDefinitions.json

```diff
@@ -200,6 +206,11 @@
             "type": "boolean",
             "description": "Whether or not it was a presigned request"
         },
+        {
+            "name": "platform",
+            "type": "string",
+            "description": "Language-specific identification"
```

Comment:
What does this represent? Doesn't seem language-specific if it's paired with a runtime?

Reply:
This is a little more open ended, but you'd get language specific values, for example:
C#: `v4.6.1`, `netcoreapp2.1`, `netcoreapp3.1`
node: `nodejs10.x`, `nodejs12.x`

There may be some overlap with Lambda runtimes, but this isn't strictly tied to lambda usage. Runtimes like `provided` (for Custom Runtimes) are opaque, so this also provides a chance to get additional detail. (this is how we would have captured .NET Core 3.0 usage)

Reply:
Can that be added to the description?

Reply:
Done